crates/hir_def/src/item_tree/tests.rs

```diff
@@ -437,3 +437,23 @@ fn assoc_item_macros() {
         "#]],
     );
 }
+
+#[test]
```

Comment:
Can you also add a test (or change this one) with an unsafe intrinsic?

Reply:
I moved the tests in a better location, and included both a safe and an unsafe intrinsic